cfme/tests/cloud/test_project_quota.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+from riggerlib import recursive_update
+from widgetastic.utils import partial_match
+
+from cfme import test_requirements
+from cfme.cloud.instance import Instance
+from cfme.cloud.provider.azure import AzureProvider
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.utils.generators import random_vm_name
+
+pytestmark = [
+    test_requirements.quota,
+    pytest.mark.long_running,
+    pytest.mark.provider([AzureProvider, OpenStackProvider],
+                         required_fields=[['provisioning', 'image']], scope="module")
+]
+
+
+@pytest.fixture
+def vm_name():
+    return random_vm_name(context='quota')
+
+
+@pytest.fixture
+def template_name(provisioning):
+    return provisioning["image"]["name"]
+
+
+@pytest.fixture(scope="module")
+def roottenant(appliance):
+    return appliance.collections.tenants.get_root_tenant()
+
+
+@pytest.fixture
+def prov_data(provider, provisioning, vm_name, template_name):
```

Comment:
This fixture is the same in the other file, consider moving to a common place and re-using it in both places?

Reply:
Yes, the fixture is common and generally vary from provider to provider. So can I add it to the fixture with provider specific return statement?

Reply:
Not sure what you are referring to by 'global fixture', I assumed you might just put it in the same folder as the tests, if these tests are the only 2 that are using it. Are you thinking somewhere else?

Reply:
I was thinking of adding it in some fixture file or creating a new one i.e cfme/fixture/quota.py
@bsquizz  any thoughts on that?

Reply:
Yeah I would say -- if it is a very general fixture you think that other tests could use, it could live in cfme/fixture. If these fixtures are really only useful for quote tests, it may actually make the most sense if both of these tests live in 1 file -- test_quota.py -- that way you can re-use the fixtures for both of the tests easily, and no need to import them from anywhere :)